tests/test_model.py

```diff
@@ -389,7 +470,28 @@ def test_serialization_by_filename(self):
         for n0, n1 in zip(model.iter_symbols(), new.iter_symbols()):
             self.assertIs(type(n0), type(n1))
 
-    def test_serialization_max_num_states(self):
+    def test_invalid_version(self):
+        from dwave.optimization._model import DEFAULT_SERIALIZATION_VERSION
```

Comment:
perhaps expose this constant in a public module?

Reply:
I think we will in future PRs. But for this PR, because we only support one version anyway, I'd like to keep it private.